### ISSUE TITLE
broken Unicode text at line #0

### ISSUE THREAD

**Issue by bsideup** [jcabi/jcabi-github#1042]
broken Unicode text at line #0 in '{"url":"https://api.github.com/repos/mycorp/..687741..\n \t\t\t\t}\n \t\t\t\treturn rules;"}]}' (688554 bytes)

**Comment by cvrebert** [comment]
@bsideup I think more details about your problem will be required.

**Comment by yegor256** [comment]
@bsideup indeed, please provide more details. how can I reproduce this problem?
